src/catalog/controller/payment/omise.php

```diff
@@ -75,7 +75,8 @@ public function checkout() {
 							"currency"    => $this->currency->getCode(),
 							"description" => $this->request->post['description'],
 							"return_uri"  => $this->url->link('payment/omise/checkoutcallback&order_id='.$order_id),
-							"card"        => $this->request->post['omise_token']
+							"card"        => $this->request->post['omise_token'],
+							"capture"     => $this->config->get('omise_auto_capture')
```

Comment:
@nimid I'm not sure, do we need to check something like https://github.com/omise/omise-opencart/pull/30/files#diff-3a51e68d7e5592d547fa719fd60d8611R154 here?

`$this->config->get('omise_auto_capture') ? $this->config->get('omise_auto_capture') : ...;`